packages/shared/src/node/env/GlobalValues.ts

```diff
@@ -7,6 +7,7 @@ export default class GlobalValues {
   public readonly isProduction = getEnv('NODE_ENV') === 'production';
   public readonly isIntegrationTest = yes(getEnv('INTEGRATION_TEST'));
   public readonly isUnitTest = getEnv('NODE_ENV') === 'test';
+  public readonly isDevFeaturesEnabled = !this.isProduction || yes(getEnv('DEV_FEATURES_ENABLED'));
```

Comment:
should we align this with console? thus you can directly use this env for checking

<img width="586" alt="image" src="https://github.com/logto-io/logto/assets/14722250/4c790d99-9a74-426d-8fee-4ddf7a1740cc">

Reply:
updated